tests/src/test/scala/whisk/core/limits/ActionLimitsTests.scala

```diff
@@ -269,4 +269,29 @@ class ActionLimitsTests extends TestHelpers with WskTestHelpers {
             }
     }
 
+    it should "limit the length of HTTP req/resp bodies for debug messages to 1000 bytes" in withAssetCleaner(wskprops) {
+        (wp, assetHelper) =>
+            val auth: Seq[String] = Seq("--auth", wskprops.authKey)
+            val name = "largeFileAction"
+            val debugMsg = "Body excedes 1000 bytes and will be truncated"
```

Comment:
If there is a large file, let's say 1MB or larger, there is a great deal of data dumping onto the screen. A truncation could help limit this enormous amount of data being displayed to the user/developer. But in smaller cases, like you pointed out, it seems to hamper the results more than it helps. This is why I've been thinking of either creating a separate flag or a raising the cap from 1000 bytes.

It's true, this issue could be prevented by using verbose where only `req body` and `code` are displayed but there is a loss of important debug features here. Also, the original issue called for the implementation of truncation with debug which is why I proceeded down this path. Since debug/verbose is used so sparingly, that brings up a concern; how sparingly would someone use debug AND have a large file?